lms/djangoapps/completion/migrations/aggregate_completion.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
@macornwell  Can you please rename this migration file to: `0002_aggregate_completion.py`?

This number prefix is not technically required, but is very useful when humans try to figure out which migrations go where.